### ISSUE TITLE
Why the learning rate is fixed

### ISSUE THREAD

**Issue by junleiz** [raminnakhli/Decoupled-Contrastive-Learning#6]
Hello, thanks for your excellent work. I noticed that your learning rate is fixed during training. Whey you do not apply a learning rate schedule?

**Comment by raminnakhli** [comment]
Hello,

Sorry that I see this late. The SimCLR implementation is an exact copy of the repository that the paper had cited. 
In any case, that's a good point! Thanks for bringing it up. I'll add it if I got the time. Also you are more than welcome to collaborate, if you wish! 😃
